app/controllers/api/v1/projects_controller.rb

```diff
@@ -23,6 +24,13 @@ def index
     render json: Api::V1::ProjectSerializer.new(paginate(@projects), @options)
   end
 
+  # GET /api/v1/projects/search?q=:query
+  def search
+    base_url = "#{api_v1_projects_search_url}?q=#{params[:q]}"
```

Comment:
```ruby
@projects =  ProjectsQuery.new(params, Project.public_and_not_forked).results
```

Reply:
I have initially implemented using this only, but it throws exceptions.

SS:-
![Screenshot from 2022-06-21 16-38-42](https://user-images.githubusercontent.com/77198905/174785736-0131d563-b837-4dd0-85b6-9540c230b979.png)

Reply:
Updated the snippet, the params format is different for website search